test/battle/ability/color_change.c

```diff
@@ -4,41 +4,46 @@
 SINGLE_BATTLE_TEST("Color Change changes the type of a Pokemon being hit by a move if the type of the move and the Pokemon are different")
 {
     GIVEN {
+        ASSUME(gSpeciesInfo[SPECIES_KECLEON].types[0] == TYPE_NORMAL || gSpeciesInfo[SPECIES_KECLEON].types[1] == TYPE_NORMAL);
```

Comment:
Isn't this assume technically wrong? If Kecleon would be a pure Ground type, the test would still be passing.

Reply:
True. Will adjust to check that it's not Psychic type